dart/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 25.9.23
+
+- use enhanced enums (#8313)
+- fix incorrect write in Float64 write method (#8290)
+- code format improvements (#8707)
```

Comment:
We could do this for example:
```suggestion
- use enhanced enums (#8313, part of GH release 24.12.13)
- fix incorrect write in Float64 write method (#8290, part of GH release 24.12.13)
- code format improvements (#8707, part of GH release 25.9.23)
```

but then it won't match the actual changelog that's pushed on pub.dev (that's part of the package). I don't have a strong opinion though.

Reply:
There's no automation so I've just updated it manually (by filtering the commits related to the relevant paths) before pushing the package.